styles/main.scss

```diff
@@ -464,6 +468,21 @@ header {
   object-fit: cover;
 }
 
+.intro-avatar .image .edit-background {
```

Comment:
I'm not sure what this element is

We don't have any elements to modify any backgrounds